app_dart/lib/src/request_handlers/refresh_chromebot_status.dart

```diff
@@ -53,8 +58,13 @@ class RefreshChromebotStatus extends ApiRequestHandler<Body> {
   Future<Body> get() async {
     final LuciService luciService = luciServiceProvider(this);
     final DatastoreService datastore = datastoreProvider(config.db);
+    final Commit latestCommit = await datastore.queryRecentCommits(limit: 1).single;
+    final SchedulerConfig schedulerConfig = await scheduler.getSchedulerConfig(latestCommit);
+    final List<Target> postsubmitTargets = scheduler.getPostSubmitTargets(latestCommit, schedulerConfig);
+    final List<LuciBuilder> postsubmitBuilders =
```

Comment:
I may miss some context. Does `postsubmitBuilders` here include those in builders.json as well, or only those from .ci.yaml? Or we will wait for targets migrated to .ci.yaml before landing this?

Reply:
Thanks, added `scheduler.getPostSubmitBuilders` to aggregate these. This will support both configs during the migration.